compiler/rustc_feature/src/active.rs

```diff
@@ -688,6 +690,10 @@ declare_features! (
     /// not changed from prior instances of the same struct (RFC #2528)
     (incomplete, type_changing_struct_update, "1.58.0", Some(86555), None),
 
+    /// Allows using internal rustdoc features like doc(primitive) or doc(keyword)
+    /// without a future-incompat warning.
```

Comment:
`doc(keyword)` on stable is a hard error, only `doc(primitive)` is a warning. Also, please use inline code ;)